mp/game/momentum/resource/momentum_english_ref.res

```diff
@@ -620,6 +620,9 @@
         "MOM_MB_Safeguard_Map_Quit_ToMenu_Title" "Quit To Menu Safeguard"
         "MOM_MB_Safeguard_Map_Quit_ToMenu_Msg" "You are trying to quit to menu while your timer is running!\nAre you sure you want to quit?\n(You can go to Settings -> Gameplay -> Safeguards to turn this off!)"
 
+        "MOM_Safeguard_Double_Press_Warning" "You must double press the key for that command\nwhile the timer is running!\nYou can turn this safeguard off in the Gameplay Settings!"
```

Comment:
I agree these should be localized though doing it this way eliminates the safeguard type from the message. I worry that this'll confuse players. Hard to think of a way that's not exhaustive though unfortunately.